pipeline/common.sh

```diff
@@ -61,7 +68,7 @@ function setupBuildBase {
   cp -R ${DOCKER_SOURCE}/setup/db ${DOCKER_SOURCE}/tests/sidecar/setup
 
   local dotcmsDockerImage=${1}
-  gitFetchRepo 'https://github.com/dotCMS/docker.git' ${dotcmsDockerImage}
+  gitFetchRepo 'https://github.com/dotCMS/docker.git' ${dotcmsDockerImage} "release-21.05"
```

Comment:
let's keep "master" in here or just remove the last parameter "release-21.05", which is equivalent of putting "master"

Reply:
Done